notebooks/build_person_directory.ipynb

```diff
@@ -0,0 +1,371 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "759f9ec0",
+   "metadata": {},
+   "source": [
+    "# Person Directory"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "61534b8b",
+   "metadata": {},
+   "source": [
+    "## Objective\n",
+    "\n",
+    " This script builds a Person Directory by enrolling multiple faces for each person from a directory where each subfolder represents a different person. It creates a person for each subfolder and adds all face images within that subfolder to the created person.\n",
+    "\n",
+    "| Enrollment | Searching |\n",
+    "| :-: | :-: |\n",
+    "| ![PD_enrollment.png](https://media.githubusercontent.com/media/Azure-Samples/azure-ai-content-understanding-python/refs/heads/zhizho/face/data/face/PD_enrollment.png) | ![PD_searching.png](https://media.githubusercontent.com/media/Azure-Samples/azure-ai-content-understanding-python/refs/heads/zhizho/face/data/face/PD_searching.png) |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "80a375ca",
+   "metadata": {},
+   "source": [
+    "## Create Azure content understanding face client\n",
+    ">The [AzureContentUnderstandingFaceClient](../python/content_understanding_face_client.py) is utility Class which contain the functions to interact with the Content Understanding face server. Before Content Understanding SDK release, we can regard it as a lightweight SDK. Fill the constant **AZURE_AI_ENDPOINT**, **AZURE_AI_API_VERSION**, **AZURE_AI_API_KEY** with the information from your Azure AI Service."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a5b1fa11",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "import os\n",
+    "import sys\n",
+    "from pathlib import Path\n",
+    "from dotenv import find_dotenv, load_dotenv\n",
+    "from azure.identity import DefaultAzureCredential, get_bearer_token_provider\n",
+    "\n",
+    "# import utility package from python samples root directory\n",
+    "parent_dir = Path(Path.cwd()).parent\n",
```

Comment:
Do we need so many casts?

```suggestion
    "parent_dir = Path.cwd().parent\n",
```

Since this is not critical, perhaps inline into sys.path.append() so users focus less on this?